src/assets/i18n.json

```diff
@@ -170,10 +170,19 @@
     "updateReady": "更新の準備ができました、このページをリロードしてください！",
     "credits": "Credits",
     "lastUpdate": "楽曲の更新日：{date}",
+    "about": "App inspired by work from Jeff Lloyd. * Weighted distribution code contributed by Chris Chike. * Jacket images prepared by FuriousDCSL. * Japanese localization made possible by nalpan. * Maintained by Noah Manneschmidt.",
     "contact": {
+      "prompt": "Please reach out with feedback, ideas, or code contributions:",
       "facebook": "Facebook Messengerで作者に連絡する",
       "twitter": "Twitterで作者に連絡する",
       "github": "Githubで貢献する"
+    },
+    "songAction": {
+      "lock": "Protect",
```

Comment:
@nalpan I added a few new strings here. I would be happy if you have the time to localize them eventually 😄